src/styles/blocks/nav.scss

```
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+  height: 60px;
+  align-items: center;

```

remove flex properties and height from nav, you've already add height to nav__item and list is flex container for these items
```suggestion
.nav {
```suggestion

---

in nav, I can not remove flex properties from nav, since it is responsible for centering and so that __link does not get out of the border